pkg/launcher/launcher.go

```diff
@@ -122,8 +136,10 @@ func Launch(params Parameters) {
 	}
 
 	// Add the Metrics Service
-	addMetrics(ctx, cfg, namespace)
+	if !strings.Contains(namespace, ",") {
```

Comment:
@rigazilla looks like we should invoke `serveCRMetrics` explicitly otherwise.

Reply:
@dmvolod `namespace` is used as a namespace later in the [code](https://github.com/infinispan/infinispan-operator/blob/3688b94a4ae0e06d94ba1d5853cc4475e96dafcb/pkg/launcher/launcher.go#L183-L184), passing the `namespace` value as is, doesn't work for multinamespace.
I didn't spent a lot of time on it since this call has been removed in [1.0.0](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.0.0/#removed-packages-pkgkube-metrics-and-pkgmetrics), I've just minimized the changes for now.

I don't know, maybe we can just replace it with the namespace where the operator is actually running?

Reply:
Yeah, sure @rigazilla . This could be `k8sutil.GetOperatorNamespace`
I agree with you, we need to move to the Operator SDK `1.x.x` soon